proco/schools/loaders/ingest.py

```diff
@@ -25,19 +27,46 @@ def load_data(uploaded_file):
     return loader.load_file(uploaded_file)
 
 
+def _find_country(file: File) -> Country:
+    loaded = load_data(file)
+
+    points = MultiPoint()
+    for _i, data in enumerate(loaded):
+        try:
+            point = Point(x=float(data['lon']), y=float(data['lat']))
+
+            if point == Point(x=0, y=0):
+                continue
+            else:
+                points.append(point)
+        except (TypeError, ValueError):
+            continue
+
+    countries = Country.objects.filter(geometry__contains=points)
+    if countries.count() > 1:
```

Comment:
let's take country having most of the points as main. this will allow us to keep boundaries validation untouched